.gitignore

```diff
@@ -29,3 +29,4 @@ yarn-debug.log*
 yarn-error.log*
 
 docker-compose.override.yml
+.storybook
```

Comment:
We don't want to ignore this directory, because that's where all the storybook configuration lives. 
What command were you running that triggered creation of the js files in your screenshot? I haven't seen that before

Reply:
Hm, I see. I'm not sure, I don't remember running anything other than `yarn storybook`, but I probably did. Anyway, I can just delete the files and pay attention if they get generated again.